ci/run-benchmarks.sh

```diff
@@ -13,6 +13,8 @@ fi
 
 # we use taskset to pin the process to a specific set of cores and their HTs to reduce benchmark result variance
 # this set is chosen to all be located on the second die of the Threadripper 2920X CPU in our current CI benchmark system "gpuc2"
+# since this is a very rare system with only 3 cores per CC, we use a hardcoded thread pinning strategy which places the 4 BE worker threads on HTs
+export CELERITY_THREAD_PINNING=7,8,9,10,11,22,23
 bash /root/capture-backtrace.sh taskset -c 6-11,18-23 test/benchmarks \
```

Comment:
No, removed. (until the results tell me otherwise at least)

Reply:
And the answer is in, we still need it. Otherwise the non-runtime benchmarks run on a different set of cores, unless either (i) we hardcode that in the benchmark pinner, or (ii) we manually interpret the environment variable in the non-runtime benchmarks.

I think both of those are worse than keeping the taskset, so I'll do that.

Reply:
Aren't we now instantiating a thread pinner for all benchmarks anyway?

Reply:
We are, I'm talking about configuring the thread pinner.
By default it would take a different core than what used to be the case.

Reply:
Mhh we don't parse the environment in non-RT tests. Makes sense.